templates/ocs-deployment/CephBlockPool.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ cephblockpool_name | default('ocsci-cephblockpool') }}
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
```

Comment:
I realize that the namespace for block pool cannot be anything other than openshift-storage. Rook looks to create this block pool in the ceph cluster in openshift-storage namespace. So if one provides any other namespace it simply fails.

  Warning  ProvisioningFailed  10s (x2 over 25s)  ceph.rook.io/block_rook-ceph-operator-5b6856f864-t67gx_18bb8827-7d15-11e9-83b8-0a580a800217  failed to provision volume with StorageClass "ocs-qe": Failed to create rook block image krk-rbd/pvc-3a4ea058-7df8-11e9-9bd6-0ac430f8eef8: failed to create image pvc-3a4ea058-7df8-11e9-9bd6-0ac430f8eef8 in pool krk-rbd of size 4294967296: Failed to complete '': exit status 1. global_init: unable to open config file from search list /var/lib/rook/krk/krk.config

please note that krk is the ceph cluster rook is trying to find and fail.

So, this value should be hardcoded with 'openshift-storage'. I'll also send a patch to replace that in the storage-manigest.yaml file.

Reply:
@keesturam, in that case, I suggest to leave it as it is with openshift-storage as the default.